packages/coinjoin/src/enums.ts

```diff
@@ -1,43 +1,47 @@
-export enum SessionPhase {
-    // RoundPhase.InputRegistration
-    RoundSearch = 101,
-    CoinSelection = 102,
-    RoundPairing = 103,
-    CoinRegistration = 104,
+export const sessionPhases = {
```

Comment:
nit: how about capitalizing it?

Reply:
`SessionPhases`? Could be, we should agree on some convention. But it feels a bit weird to use PascalCase for a JS object.

Reply:
True but it's also weird to use camelCase for something that acts like and Enum 😓 The values fields are still PascalCase, right? `roundPhases.OutputRegistration`

Reply:
Went back to PascalCase and singular: 8633806301aa9e24b4e9fe042e8defd95dcd76e4